discord/guild.py

```diff
@@ -48,6 +48,8 @@
 )
 import warnings
 
```

Comment:
discord.types references need to be inside `if TYPE_CHECKING` blocks. This should also probably be aliased as `GuildShopPayload` to avoid a clash with the (currently non-existent) discord.py representation.

Reply:
Yep, I was going to check that but in mobile when editing big files it is a bit buggy

Reply:
Changes as you said.